swarmer/src/main/kotlin/com/gojuno/swarmer/Emulators.kt

```diff
@@ -334,3 +347,17 @@ private fun outputDirectory(args: Commands.Start) =
 
 private fun connectedEmulators(): Single<Set<AdbDevice>> =
         connectedAdbDevices().take(1).toSingle().map { it.filter { it.isEmulator }.toSet() }
+
+private fun os(): Os {
```

Comment:
We can expose this from Commander https://github.com/gojuno/commander/blob/05d1714151ae98169fcfd300bf55ffc2f93171d0/os/src/main/kotlin/com/gojuno/commander/os/Processes.kt#L129

Reply:
Lets do it, sounds good

Reply:
https://github.com/gojuno/commander/pull/23